packages/core/src/routes/experience/classes/experience-interaction.ts

```diff
@@ -43,6 +45,8 @@ type InteractionStorage = {
   profile?: InteractionProfile;
   mfa?: MfaData;
   verificationRecords?: VerificationRecordData[];
+  captchaVerified?: boolean;
+  captchaSkipped?: boolean;
```

Comment:
nit: prefer to wrap it in a single object filed like mfa.  
captcha: {
 verified: boolean;
 skipped: boolean;
}